services/tracks/src/services/trackService.js

```diff
@@ -3,8 +3,22 @@
 const { Tracks } = require('../db/models/Tracks');
 const { v4: uuid } = require('uuid');
 const { axiosClient } = require('../config/axios.config');
+const { kafkaTopics } = require('../constants');
+
+class TrackService {
+  notificationProducer;
+
+  constructor(notificationProducer) {
+    this.notificationProducer = notificationProducer;
+    (async () => {
```

Comment:
I would rather make this IIFE as a class method and call it `setup` or smth, looks strange for me